run.py

```diff
@@ -86,7 +86,7 @@ def create_tables():
 
     # uncomment the line below if no dummy data needed on INITIAL setup!
     # !!! Warning !!! Do not uncomment if this is not your INITIAL setup to database!
-    db.create_all()
```

Comment:
should I just remove them? line 67, 94 and all the warnings (including warnings for line 97). Their purpose are actually to remind me not to mess up with the Heroku elephantsql in case I accidentally create a new heroku image from the branch PR for remote testing. It won't have affect to anyone else since they have no access to the remote heroku. 🤣

Reply:
now that we've done the mock data test on basic setup, I don't think line 67, 93-94 are needed anymore. will remove them now.